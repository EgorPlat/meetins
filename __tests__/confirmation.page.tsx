import React from "react";
import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";
import Confirmation from "../pages/confirmation/index";
 
global.React = React;

it("Snapshot for Confirmation page is valid", () => {
    const { container } = render(<Confirmation />)
    expect(container).toMatchSnapshot()
})
  