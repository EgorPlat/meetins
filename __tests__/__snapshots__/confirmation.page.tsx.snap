// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot for Confirmation page is valid 1`] = `
<div>
  <div>
    <div>
      <img
        alt="Logo"
        height="50"
        src="[object Object]"
        width="50"
      />
    </div>
    <div>
      Пожалуйста, для успешной процедуры подтверждения не перезагружайте страницу. На указанную Вами при регистрации почту был выслан код подтверждения. Пожалуйста впишите его в поле и нажмите "Отправить"
    </div>
    <div>
      <div>
        <input
          id="customSplitedItem0"
          maxlength="1"
          placeholder=""
        />
        <input
          disabled=""
          id="customSplitedItem1"
          maxlength="1"
          placeholder=""
          style="border: 1px solid gray;"
        />
        <input
          disabled=""
          id="customSplitedItem2"
          maxlength="1"
          placeholder=""
          style="border: 1px solid gray;"
        />
        <input
          disabled=""
          id="customSplitedItem3"
          maxlength="1"
          placeholder=""
          style="border: 1px solid gray;"
        />
        <input
          disabled=""
          id="customSplitedItem4"
          maxlength="1"
          placeholder=""
          style="border: 1px solid gray;"
        />
        <input
          disabled=""
          id="customSplitedItem5"
          maxlength="1"
          placeholder=""
          style="border: 1px solid gray;"
        />
      </div>
    </div>
    <div>
      <button>
        Отправить
      </button>
      <a
        href="/login"
      >
        Вернуться назад
      </a>
    </div>
  </div>
</div>
`;
